app/client/components/billing/invoicesTable.tsx

```diff
@@ -0,0 +1,328 @@
+import { css } from "@emotion/core";
+import { palette, space } from "@guardian/src-foundations";
+import { headline, textSans } from "@guardian/src-foundations/typography";
+import moment from "moment";
+import React, { useState } from "react";
+import { DATE_INPUT_FORMAT, formatDateStr } from "../../../shared/dates";
+import { InvoiceDataApiItem } from "../../../shared/productResponse";
+import { maxWidth, minWidth } from "../../styles/breakpoints";
+import { Pagination } from "../pagination";
+import { CardDisplay } from "../payment/cardDisplay";
+import { DirectDebitDisplay } from "../payment/directDebitDisplay";
+import { PayPalDisplay } from "../payment/paypalDisplay";
+import { DownloadIcon } from "../svgs/downloadIcon";
+import { InvoiceTableYearSelect } from "./invoiceTableYearSelect";
+
+const invoicePaymentMethods = {
+  CARD: "card",
+  DIRECT_DEBIT: "directdebit",
+  PAYPAL: "paypal"
+};
+
+interface InvoiceInfo extends InvoiceDataApiItem {
+  product?: string;
+  currencyISO: string;
+  currency: string;
+}
+
+interface InvoicesTableProps {
+  resultsPerPage: number;
+  invoiceData: InvoiceInfo[];
+}
+
+export const InvoicesTable = (props: InvoicesTableProps) => {
+  const initialPage = 1;
+
+  const [currentPage, setCurrentPage] = useState<number>(initialPage);
+
+  const tableHeadings = ["Date", "Payment method", "Price", ""];
+  const invoiceYears = [
+    ...new Set(
```

Comment:
does IE support `Set`? (or are sure it's being polyfilled)

Reply:
ie11 seems to have enough support: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set#Browser_compatibility

![Screenshot 2020-08-24 at 11 20 47](https://user-images.githubusercontent.com/2510683/91034601-2ba5e380-e5fd-11ea-9690-682b37ae4239.png)